db/data/seedDatabase.js

```diff
@@ -1,6 +1,9 @@
 const makeFakeRestaurant = require('./fakeDataGenerator.js');
 const { MongoClient } = require('mongodb');
 
+const batchSize = parseInt(process.env.BATCH_SIZE, 10) || 10000;
+const numBatches = parseInt(process.env.BATCH_SIZE, 10) || 1000;
```

Comment:
I think you want this to be `NUM_BATCHES` or something along those lines! otherwise these are both referring to the same variable! @davidseid